File: Code/Tier3-DatabaseLayer/RestServer/Common/SprintList.cs
using System.Collections.Generic;

namespace WebApplication.Common
{
    public class SprintList

    {
        public int duration{ get; set; }
        public List<Sprint> sprintsList{ get; set; }
        public int numberOfSprints{ get; set; }
        public int groupID{ get; set; }
        public int currentlyActiveSprint{ get; set; }
        public bool validated { get; set; }

 
        public SprintList()
        {
        }

        public SprintList(int duration)
        {
            this.duration = duration;
            sprintsList = new List<Sprint>();
        }

   
      
    }
}

File: Code/Tier2/Common/src/main/java/javaModel/SprintList.java
package javaModel;

import java.io.Serializable;
import java.util.ArrayList;

public class SprintList implements Serializable {

    public int duration;
    public ArrayList<Sprint> sprintsList;
    public int numberOfSprints;
    public int groupID;
    public int currentlyActiveSprint;
    private boolean validated;

    public boolean isValidated() {
        return validated;
    }

    public void setValidated(boolean validated) {
        this.validated = validated;
    }

    public int getGroupID() {
        return groupID;
    }

    public void setGroupID(int groupID) {
        this.groupID = groupID;
    }

    public SprintList ()
    {

    }
    public SprintList(int duration){
        this.duration = duration;
        sprintsList = new ArrayList<>();
    }

    public void setDuration(int duration){
        this.duration = duration;
    }

    public int getDuration(){
        return  duration;
    }

    public ArrayList<Sprint> getSprintsList() {
        return sprintsList;
    }

    public void setSprintsList(ArrayList<Sprint> sprintsList) {
        this.sprintsList = sprintsList;
    }

    public int getNumberOfSprints() {
        return numberOfSprints;
    }

    public void setNumberOfSprints(int numberOfSprints) {
        this.numberOfSprints = numberOfSprints;
    }

    public int getCurrentlyActiveSprint() {
        return currentlyActiveSprint;
    }

    public void setCurrentlyActiveSprint(int currentlyActiveSprint) {
        this.currentlyActiveSprint = currentlyActiveSprint;
    }


}
